README.md

```diff
@@ -100,29 +102,13 @@ class App extends Component {
 export default App;
 ```
 
-4- Customize the widget to match your app design! You can add both props to manage the title of the widget, the avatar it will use and of course the styles it will have.
+4- Customize the widget to match your app design! You can add both props to manage the title of the widget, the avatar it will use and of course, you can change, from the CSS, the styles the widget will have.
```

Comment:
there are a lot of commas at the end of this sentence, check how to paraphrase it with iniaki